### DISCUSSION TITLE
tanya terkait bikin transparent proxy di aws

### DISCUSSION THREAD

**Discussion by tarecha** [awsugid/forum#38] [Q&A]
Permisi pak .
saya mau ada tugas akhir, rencananya bikin squid transparent proxy di aws.
mohon info apakah TOS AWS memperbolehkan proxy server ?
apakah ada produk yang paling murah ? setahu saya ini semacam VPS EC2
apakah itu dapat IP public yang dapat di tembak oleh router

**Comment by rioastamal** [comment]
@tarecha harusnya tidak masalah. Bisa pakai Amazon Lightsail untuk yang paling murah dan mudah karena proxy tidak terlalu butuh processing power besar.

**Reply by tarecha** [reply]
terima kasih informasinya

**Comment by saiqulhaq** [comment]
@tarecha jadinya pake EC2 atau Lightsail? jadinya kena charge internet bandwidth usage ya?

**Comment by tarecha** [comment]
fyi saya sudah buat lightsail. tapi ada notifikasi tidak boleh dibuat proxy. jadi saya hapus lagi
